src/services/ros/realms.ts

```diff
@@ -82,6 +87,31 @@ export const open = async (params: {
   return realm;
 };
 
+const getPathOnDisk = (
```

Comment:
I think it's not trivial why this rewrite is happening - and especially not why its happening only on Windows. Perhaps you could add a comment and optionally a link to the issue on GitHub?